README.md

```diff
@@ -171,16 +173,33 @@ mstorage.list(){result, error in
         // do something
     }
 }
-// You can also use a Dictionary object for listing options as follows.
-// The available options are "limit", "after" and "before".
+```
+
+* With options
+
+You can also use a `Dictionary` object for listing options as follows.
+The available options are `limit`, `after` and `before`.
+
+```swift
 mstorage.list(["limit": "25", "after": "<media_id>"]){result, error in
     // do something
 }
 ```
 
-### Delete
+* Search
+
+Search media ids by user metadata
+
+```swift
+mstorage.list(["limit": "25", "after": "<media_id>",
```

Comment:
In media-storage-js, they have two examples of `list` method; one with a `Map` (with "limit" key) without label for listing, one with a `Map` (with usermeta keys) with "filter" label for searching. I don't know how they're gonna combine the two different parameters. But my point is that we should make something more similar to the JS version, not java version, and they might not use a nested Map. So maybe we should use two independent `Dictionary`, one with no label, the other with a `filter` label if possible.